mantra-common/substreams.yaml

```diff
@@ -0,0 +1,43 @@
+specVersion: v0.1.0
+package:
+  name: mantra_common
+  version: v0.0.1
+  url: https://github.com/streamingfast/substreams-foundational-modules/mantra-common
+
+network: mantra-mainnet
+
+imports:
+  injective: https://spkg.io/streamingfast/injective-common-v0.2.3.spkg
+
+modules:
+  - name: all_events
+    use: injective:all_events
```

Comment:
One thing that I am thinking, is to create a `cosmos:all_events`, `cosmos:index_events`, etc. and have the `injective _common` and the `mantra_common` import it like here.